cmd/gardenlet/app/app.go

```diff
@@ -480,6 +489,142 @@ func (g *garden) cleanupOrphanedExtensionsServiceAccounts(ctx context.Context, g
 	return flow.Parallel(taskFns...)(ctx)
 }
 
+const (
+	grmFinalizer           = "resources.gardener.cloud/gardener-resource-manager"
+	tempSecretLabel        = "resources.gardener.cloud/temp-secret"
+	tempSecretOldNameLabel = "resources.gardener.cloud/temp-secret-old-name"
+)
+
+// TODO(dimityrmirchev): Remove this code after v1.84 has been released.
```

Comment:
To be on the safe side, let's keep it for a little longer (usually we do 3 releases)

```suggestion
// TODO(dimityrmirchev): Remove this code after v1.87 has been released.
```